core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -26,8 +26,8 @@
       <login-required-message></login-required-message>
     </div>
     <div class="oppia-login-required" ng-if="$ctrl.userDetailsLoading || $ctrl.userIsLoggedIn">
-      <opportunities-list load-opportunities="$ctrl.loadContributions"
-                          on-click-action-button="$ctrl.onClickViewSuggestion"
+      <opportunities-list [load-opportunities]="$ctrl.loadContributions"
+                          click-action-button="$ctrl.onClickViewSuggestion"
```

Comment:
Why was the "on-" prefix deleted?

Reply:
Because it is recommended by angular style guide (I think). https://angular.io/guide/styleguide#dont-prefix-output-properties

Reply:
Shouldn't it be `(clickActionButton)` then?

Reply:
Yeah, I put that in my review https://github.com/oppia/oppia/pull/11850#discussion_r581580514.

Reply:
Ah ok. Thanks!